package.json

```diff
@@ -49,17 +49,17 @@
     "es6-symbol": "^3.1.1",
     "escape-html": "^1.0.3",
     "express": "^4.15.2",
-    "extract-text-webpack-plugin": "^2.1.2",
+    "extract-text-webpack-plugin": "^3.0.0",
```

Comment:
- [Release Release v3.0.0 · webpack-contrib/extract-text-webpack-plugin](https://github.com/webpack-contrib/extract-text-webpack-plugin/releases/tag/v3.0.0)

Support for webpack v2.x has been dropped.